### PR TITLE
feat(server): adding phone-home functionality: update terraform and G…

### PR DESCRIPTION
**Pull request by mshidlov**
Issue Number: #2486

## PR Details

adding phone-home functionality to amplication-server: update terraform and GitHub actions

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-server/src/util/sendServerLoadEvent.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        nodeVersion: NODE_VERSION,
+        osName: OS_NAME,
+        osVersion: OS_VERSION,
+        hostName: HOST_NAME,
+        timezone: TIMEZONE
+      },
+      timestamp: new Date()
+    };
+
+    console.dir(data);
```
Comment (yuval-hazaz) [inline packages/amplication-server/src/util/sendServerLoadEvent.ts:42]: please remove

File: packages/amplication-server/src/util/sendServerLoadEvent.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+export const sendServerLoadEvent = (): void => {
+  void getServerId().then(runtimeId => {
+    const data = {
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      api_key: POSTHOG_ID,
+      event: 'server-load-event',
+      properties: {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        distinct_id: runtimeId,
```
Comment (yuval-hazaz) [inline packages/amplication-server/src/util/sendServerLoadEvent.ts:30]: This value is not shown in posthog

File: terraform/modules/env_baseline/main.tf [hunk 3]
```diff
@@ -305,6 +305,11 @@ resource "google_cloud_run_service" "default" {
           value = var.github_app_installation_url
         }
 
+        env {
+          name  = "DISABLE_EVENT_TRACKING"
+          value = 0
```
Comment (yuval-hazaz) [inline terraform/modules/env_baseline/main.tf:310]: I think we should disable this event for our deployments - not to be mixed with real data

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
please see my comments

File: terraform/modules/env_baseline/main.tf (hunk 3 above)
Comment (mshidlov) [reply terraform/modules/env_baseline/main.tf:310]: sure this was for testing before I will merge I will change it to 1

File: packages/amplication-server/src/util/sendServerLoadEvent.ts (hunk 1 above)
Comment (mshidlov) [reply packages/amplication-server/src/util/sendServerLoadEvent.ts:42]: Thanks

**Pull request merged**